assets/src/application/ui/input/Stepper/buttons/Next.tsx

```diff
@@ -11,7 +11,11 @@ interface Props extends ButtonProps {
 const Next: React.FC<Props> = ({ isDisabled, onClick, skippable, ...props }) => {
 	const buttonText = props.buttonText || __('Next');
 	const buttonType = props.buttonType || ButtonType.PRIMARY;
-	const rightIcon = memo(skippable ? () => <ChevronDoubleRight size='small' /> : () => <ChevronRight size='small' />);
+	const rightIcon = memo(
+		skippable
+			? () => <ChevronDoubleRight size='smaller' />
+			: () => <ChevronRight size='smaller' />
+	);
```

Comment:
That `skippable` should actually be inside the component.
```suggestion
	const rightIcon = memo( () => skippable ? <ChevronDoubleRight size='smaller' /> : <ChevronRight size='smaller' /> );
```